### DISCUSSION TITLE
How to pass remove_filter param to the time filter

### DISCUSSION THREAD

**Discussion by moathOSA** [apache/superset#19660] [Q&A / Help]
how can i pass this param "remove_filter" to the time filter 
```
time >= '{{ from_dttm }}' AND derived_tstamp < '{{ to_dttm }}'
```

like this one 
```
({{ "'" + "','".join(filter_values('country', remove_filter=True)) + "'" }})
```

should we add it as enhancement or there is a way to do it ? 
cuz currently the time filter applied twice when using "Time-series" charts since it's required a temporal column

**Comment by moathOSA** [comment]
@villebro  @srinify  can you help please ??

**Comment by vinhhap** [comment]
I'm facing the same problem. Has anyone came up with a solution?

**Comment by kamelhouchat** [comment]
I encounter the same problem; is there a solution?
